src/OrderBy/OrderByDirective.php

```diff
@@ -70,11 +77,17 @@ public function manipulateArgDefinition(
         FieldDefinitionNode &$parentField,
         ObjectTypeDefinitionNode &$parentType
     ): void {
-        if ($allowedColumns = $this->directiveArgValue('columns')) {
+        if (($allowedColumns = $this->directiveArgValue('columns')) || ($allowedColumnsEnumName = $this->directiveArgValue('columnsEnum'))) {
```

Comment:
Too much happens in this one line.

Also, let's `throw new DefinitionException` when both arguments are set.

Reply:
Refactored